backend/src/main/java/com/huellapositiva/application/controller/VolunteerApiController.java

```diff
@@ -58,10 +57,9 @@
                     )
             }
     )
-
-
-    @PostMapping("/register")
+    @PostMapping
     @ResponseBody
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
To meet REST and HTTP standards if we return 201 the convention states that we should send back the URI ("/volunteer/id") of the created resource in the Location header of the response